docs/cudf/source/basics/basics.rst

```diff
@@ -15,32 +15,61 @@ The following table lists all of cudf types. For methods requiring dtype argumen
+    | Boolean                |                              | np.bool_                                                                            | ``'bool'``                                  |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Decimal                | Decimal64Dtype               | (none)                                                                              | (none)                                      |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | List                   | ListDtype₁.₁                 | np.list_                                                                            | (none):sub:``1.2``                          |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Struct                 | StructDtype₁.₁               | np.struct_                                                                          | (none):sub:``1.2``                          |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
 
 **Note: All dtypes above are Nullable**
 
+**1.1 The complete datatypes for both lists and structs are inferred from the data that it is composed upon. For example, the data type for the value** ``[[1, 2], [3, 4]]`` **is inferred as** ``ListDtype(int64)``.
+
+**1.2 cuDF does not support string aliases where dtype would equal** ``'list'`` **or** ``'struct'`` **list's and structs are inferred by the datatype explicitly or through a determination based on the data that the list or struct is composed upon**
+
+**cuDF supports arbitrarily deep nested lists. Such as** ``list(list(int))``, **even list of structs or structs of lists**
+
+**cuDF also supports arbitrary fields for structs - that is, it is possible to have a struct with any number of fields and any number of types that cuDF supports, even a struct of structs. Structs should be made up by same type of datatype or cuDF will produce an error - Example below**
+
+
+.. code-block:: python
+
+    >>> Test_struct_dtype = [{'a':'dog', 'b':'cat', 'c':'astronomy'}, {'a':'fish', 'b':'gerbil', 'c':7}]
+    >>> df = cudf.DataFrame({'Example':Test_struct_dtype})
+
+**Structs should also made up by with the same number of elements in each row or cuDF will default to creating empty values to rows - Example below**
```

Comment:
Structs require all rows have every field. We should stick to this terminology as `field` is a term of art here. It will also create null values, rather than empty.

I think these examples should also be on Series, dtypes are on a column basis.

Reply:
Agreed, will be resolved in the next commit